enki/PhysicalEngine.h

```diff
@@ -448,6 +460,8 @@ namespace Enki
 		
 		//! All the objects in the world
 		Objects objects;
+		//! A list to defined object for which the physics will be skipped
+		Objects SkipPhysicsObjectsContainer;
```

Comment:
I understand that this is to avoid having the object under drag to affect the physics. Did you think about simply using `World::addObject()` and `World::removeObject()` and then draw the object under drag by the viewer? It would pollute less the core of the engine with viewer-specific features. Moreover, in another project we are planning to add kd-tree and grid-based lookup to improve collision, and this exception list would make this feature a little bit more heavy.